### PR TITLE
Index equipments

### PR DESCRIPTION
**Pull request by erik0686**
### What does this PR do?

* Creates a partial that is rendered on the landing page, it shows all the equipment on the landing page as cards. 

* In the labs CSS stylesheet, I took out the EquipoGrid style out of the Lab Detail scope, to reuse the style in the new partial.

#### Merge Checklist:

- [x] The branch is up-to-date (i.e. rebased) with the base branch
- [x] Is GPA high enough?

### CODE REVIEW

File: app/views/shared/_equipment_landing.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div class="EquipoGrid">
```
Comment (technical-boy) [inline app/views/shared/_equipment_landing.html.erb]: Is the shared/ directory the right place for this partial?

Comment (erik0686) [reply app/views/shared/_equipment_landing.html.erb]: I thought about that, my logic was that this partial might be used later for other places where the equipments are shown... What do you think?

**Comment by erik0686** [comment]
@AldoCervantes I do not agree on adding the "Lista de equipos" all the way in the bottom, if there are too many equipments, It will be hard to see that link. So I will only add a Label that says equipos right below the lista de equipos link

File: app/views/shared/_equipment_landing.html.erb (hunk 1 above)
Comment (technical-boy) [reply app/views/shared/_equipment_landing.html.erb]: I think the shared directory is for things that appear in most or every view (i.e. flash messages, navigation elements, etc.). This partial should probaly go in views/equipment or something like that.

File: app/views/shared/_equipment_landing.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <div class="row">
+    <% @equipments.each do |e| %>
+        <div class="col-4">
+          <div class="EquipoCard" data-id="<%= e.id %>" data-lid="<%= e.lab_space.lab.id %>" data-lsid="<%= e.lab_space.id %>">
+            <div class="icon printer center-icon"></div>
+            <p class="name"><%= link_to e.name, lab_lab_space_equipment_path(e.id, e.lab_space.id, e.lab_space.lab.id) %></p>
```
Comment (DiegoAstiazaran) [inline app/views/shared/_equipment_landing.html.erb]: I think `e.lab_space.id` and `e.lab_space.lab.id` are not in the correct order.

Comment (erik0686) [reply app/views/shared/_equipment_landing.html.erb]: Nice

**Pull request merged**